LOCAL_DEVELOPMENT_SETUP.md

```diff
@@ -223,7 +223,7 @@ NB: if you are on an Apple Silicon Mac you will have to [build the base image's
 Next, launch a shell inside the container:
 
 ```
-docker run --rm -it -v $PWD:/host -w /host --network="host" --entrypoint sh registry.gitlab.com/finestructure/swiftpackageindex:be565ca16725d4836efbb2517fd0285fddbe9da0
+docker run --rm -it -v $PWD:/host -w /host --network="host" --entrypoint sh registry.gitlab.com/finestructure/spi-base:0.14.1
```

Comment:
We probably want to keep the sha because it's the reference used in the context of this example. `make docker-build` will log the sha as the image reference instead of a tag.

```suggestion
docker run --rm -it -v $PWD:/host -w /host --network="host" --entrypoint sh registry.gitlab.com/finestructure/spi-base:be565ca16725d4836efbb2517fd0285fddbe9da0
```